app/src/main/java/com/arctouch/codechallenge/util/MovieImageUrlBuilder.kt

```diff
@@ -1,17 +1,17 @@
 package com.arctouch.codechallenge.util
 
-import com.arctouch.codechallenge.api.TmdbApi
+import com.arctouch.codechallenge.BuildConfig
 
 private val POSTER_URL = "https://image.tmdb.org/t/p/w154"
```

Comment:
Since `POSTER_URL` and `BACKDROP_URL` are both constants, what do you think to enclose them in a companion object and make them `const` as well? It should be like this:

    class MovieImageUrlBuilder {
        companion object {
            private const val POSTER_URL = "https://image.tmdb.org/t/p/w154"
            private const val BACKDROP_URL = "https://image.tmdb.org/t/p/w780"
        }
        // the class body goes here
    }

Reply:
Yes, I would do this but I did not arrive to refactor this class